Fix css

lhsarmiento opened the pull request:
## Describe your changes

☻ Improvements to the CSS styles
☻ A title is added to the calculator
☻ A description is added in Readme.md

## Issue ticket number
TC52587
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

lhsarmiento committed:
Fix CSS styles

lhsarmiento committed:
fix Readme.md

oscarlonza reviewed (changes requested):
No es necesario usar el título de cuarto nivel para desarrollar la idea. Por ejemplo los nombres de los integrantes está con #### ahí no sería necesario. Puedes colocar el texto normal.

oscarlonza reviewed (approved):
Con los ajustes propuestos en el css se ve mejor prestada la distribución de botones en la calculadora 👍

oscarlonza committed:
Update README.md

Se realizan algunos cambios en archivo readme para que la visualización sea mejor.
